projects/extension/sql/incremental/020-migrate-existing-vectorizers-to-loading.sql

```diff
@@ -0,0 +1,41 @@
+do language plpgsql $block$
+DECLARE
+    _vectorizer RECORD;
+    _chunking jsonb;
+    _chunk_column text;
+    _config jsonb;
+BEGIN
+    -- Loop through all vectorizers
+    FOR _vectorizer IN SELECT id, config FROM ai.vectorizer
+    LOOP
+        -- Extract the chunking config and chunk_column
+        _chunking := _vectorizer.config operator(pg_catalog.->)'chunking';
+        _chunk_column := _chunking operator(pg_catalog.->>)'chunk_column';
+        
+        IF _chunk_column IS NOT NULL THEN
+            -- Create new config:
+            -- 1. Add loading config
+            -- 2. Add parsing config
+            -- 3. Remove chunk_column from chunking config
+            _config := _vectorizer.config operator(pg_catalog.||) jsonb_build_object(
+                'loading', json_object(
+                    'implementation': 'column',
+                    'config_type': 'loading',
+                    'column_name': _chunk_column,
+                    'retries': 6
+           ),
+                'parsing', json_object(
+                    'implementation': 'auto',
+                    'config_type': 'parsing'
+                ),
+                'chunking', _chunking operator(pg_catalog.-) 'chunk_column'
+            );
+            
+            -- Update the vectorizer with new config
+            UPDATE ai.vectorizer 
```

Comment:
We should update the `version` in the config to be the version of this release.

Reply:
For the record, happy to apply this change if that's what we want, but currently tests are failing due to the previously mentioned script overwriting our intention:

![image](https://github.com/user-attachments/assets/abfaf6d2-ecf9-4ff5-a6c7-0925a43297bb)

Reply:
ping @cevian @jgpruitt, as this one I wasn't entirely sure about and might be relevant.